pkg/kubelet/cm/qos_container_manager_linux.go

```diff
@@ -95,8 +95,8 @@ func (m *qosContainerManagerImpl) Start(getNodeAllocatable func() v1.ResourceLis
 
 	// Create containers for both qos classes
 	for qosClass, containerName := range qosClasses {
-		// get the container's absolute name
-		absoluteContainerName := CgroupName(containerName)
+		// get the container's abstract name
```

Comment:
Why change this name? This var is mean to "absolute path of cgroup name"

Reply:
It is only a conversion to CgroupName. I think it maybe typo. Do you think?

```
// CgroupName is the abstract name of a cgroup prior to any driver specific conversion.
type CgroupName string
```

Reply:
I'm afraid not, it is literally the absolute path of cgroup, please leave it.

Reply:
OK. Tks